### PR TITLE
Remove paid AMI flag and finalize ROSA transition

### PR DESCRIPTION
**Pull request by vkareh**
Since we're now moving paid AMIs to be a feature flag in Unleash, we
remove the attribute from here and let OCM make the decision based on
the user's organization.

We also finalize the transition from MOA to ROSA, since all dependencies
are now deployed.

### CODE REVIEW

**Comment by vkareh** [comment]
/hold

Needs https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/merge_requests/2341 to be promoted to prod before merging.

**Review by jharrington22 — APPROVED** [review]
Looks good with the CS MR is ready.

**Comment by vkareh** [comment]
@jharrington22 - any chance that we can add the `rosa` label to a production clusterimageset? It's blocking this change as well as the counterpart in CS

**Pull request merged**
